src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/EventGridExtensions.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Microsoft.Azure.EventGrid.Models;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
+{
+	/// <summary>
+	/// 
+	/// </summary>
+	public static class EventGridEventExtensions
+    {
+		private const string SyncTokenPropertyName = "syncToken";
+
+		/// <summary>
+		/// This method uses an EventGridEvent from EventGrid and tries to create a <see cref="PushNotification"/>
```

Comment:
>/// This method uses an EventGridEvent from EventGrid and tries to create a <see cref="PushNotification"/> [](http://example.com/codeflow?start=2&length=106)

Suggested method summary:
"Tries to create the <see cref="PushNotification"/> object from the details in <see cref="EventGridEvent"/>object. Return value indicates whether the operation succeeded or failed."

Reply:
This comment still needs to be addressed.